test/proper_tests.erl

```diff
@@ -858,7 +858,8 @@ true_props_test_() ->
      {timeout, 20, ?_passes(symb_statem:prop_parallel_simple())},
      {timeout, 10, ?_passes(ets_statem:prop_ets())},
      {timeout, 20, ?_passes(ets_statem:prop_parallel_ets())},
-     {timeout, 20, ?_passes(pdict_fsm:prop_pdict())}].
+     {timeout, 20, ?_passes(pdict_fsm:prop_pdict())}],
+     ?_passes(lists_statem:prop_simple()).
```

Comment:
This test seems a bit wrongly placed.  Notice that all others are inside the list, while this one is outside.  Can you place it before or after the line which reads: `?_passes(pdict_statem:prop_pdict())`  (if you have such a line in your fork)?